### PR TITLE
OSDOCS-7517: 4.13 ZTP manifests

### PR DESCRIPTION
**Pull request by skopacz1**
[OSDOCS-7517](https://issues.redhat.com/browse/OSDOCS-7517)

Version: 4.13

This is the 4.13 implementation/backport of #63873 

QE review:
- [x] QE has approved this change.

Previews:

- [Creating the preferred configuration inputs](https://64085--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_with_agent_based_installer/installing-with-agent-based-installer#installing-ocp-agent-inputs_installing-with-agent-based-installer)
- [Optional: Using ZTP manifests](https://64085--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_with_agent_based_installer/installing-with-agent-based-installer#installing-ocp-agent-ztp_installing-with-agent-based-installer)
- [Creating and booting the agent image](https://64085--docspreview.netlify.app/openshift-enterprise/latest/installing/installing_with_agent_based_installer/installing-with-agent-based-installer#installing-ocp-agent-boot_installing-with-agent-based-installer)

### CODE REVIEW

File: modules/installing-ocp-agent-inputs.adoc
Comment (skopacz1) [inline modules/installing-ocp-agent-inputs.adoc:1]: These steps were moved out of modules/installing-ocp-agent-boot.adoc and the module is now parallel to how the 4.14 docs are structured: [Creating the preferred configuration inputs](https://docs.openshift.com/container-platform/4.14/installing/installing_with_agent_based_installer/installing-with-agent-based-installer.html#installing-ocp-agent-inputs_installing-with-agent-based-installer)

**Comment by skopacz1** [comment]
@bmanzari could you PTAL at this PR? It's the 4.13 implementation of the #63873 PR that you reviewed. Thanks!

File: modules/installing-ocp-agent-inputs.adoc [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+.Procedure
+
+. Install `nmstate` dependency by running the following command:
++
+[source,terminal]
+----
+$ sudo dnf install /usr/bin/nmstatectl -y
+----
+
+. Place the `openshift-install` binary in a directory that is on your PATH.
```
Comment (cbippley) [inline modules/installing-ocp-agent-inputs.adoc:20]:
```suggestion
. Place the `openshift-install` binary in a directory that is on your `PATH`.
```
I think PATH should be in backticks.

**Review by cbippley — COMMENTED** [review]
One nit but LGTM!

**Comment by cbippley** [comment]
As I am still in training, please wait for @opayne1 to review before requesting a merge review. Thank you!
/label peer-review-done
/remove-label peer-review-in-progress

File: modules/installing-ocp-agent-inputs.adoc (hunk 1 above)
Comment (skopacz1) [reply modules/installing-ocp-agent-inputs.adoc:20]: I can't believe I keep missing this one, good catch!  🤦‍♂️

**Review by opayne1 — COMMENTED** [review]
Sorry this took me so long to get back to 

/lgtm

**Pull request merged**